benchopt/stopping_criterion.py

```diff
@@ -72,7 +72,10 @@ def __init__(self, strategy=None, key_to_monitor='objective_value',
 
         self.kwargs = kwargs
         self.strategy = strategy
-        self.key_to_monitor = key_to_monitor
+        self.key_to_monitor = (
```

Comment:
this does another thing that is not related to output processing/saving, do you want it merged in this PR?

Reply:
Yes, let's not overcomplicate the process ^^
I usually try to open separate PR but sometimes I take some shortcuts, sorry about that